lib/motions/general-motions.coffee

```diff
@@ -140,9 +140,9 @@ class MoveLeft extends Motion
   operatesInclusively: false
 
   moveCursor: (cursor, count=1) ->
-    _.times count, ->
-      unless cursor.isAtBeginningOfLine()
-        cursor.moveLeft()
+    _.times count, =>
+      cursor.moveLeft() unless cursor.isAtBeginningOfLine() and !atom.config.get('vim-mode.wrapLeftRightMotion')
```

Comment:
This looks right, but the combination of `unless` and `!` is a little confusing to me. Could you switch this to use an `if` instead?